app/javascript/crayons/Modal/__stories__/modals.mdx

```diff
@@ -29,13 +33,13 @@ The "special styling" means it will:
 
 ### Centered prompt
 
-Some prompts may visually benefit from centering the content. 
+Some prompts may visually benefit from centering the content.
 Leave the decision to designers but keep it mind this is possible with `centered` prop
 (and it will only work in combination with `prompt`).
 
 ## Backdrop
 
-By default Modals come with dark backdrop (a layer overlaying underlaying content). 
+By default Modals come with dark backdrop (a layer overlaying underlaying content).
```

Comment:
Quick, non-blocking suggestion for clarity purposes:
```suggestion
By default, Modals come with a dark backdrop (a layer overlaying underlaying content).
```

Reply:
Thanks for these fixes! 🎉